tf2_ros/include/tf2_ros/message_filter.h

```diff
@@ -32,690 +32,550 @@
-{
-enum FilterFailureReason
-{
-  /// The message buffer overflowed, and this message was pushed off the back of the queue, but the reason it was unable to be transformed is unknown.
-  Unknown,
-  /// The timestamp on the message is more than the cache length earlier than the newest data in the transform cache
-  OutTheBack,
-  /// The frame_id on the message is empty
-  EmptyFrameID,
-};
-}
-typedef filter_failure_reasons::FilterFailureReason FilterFailureReason;
-
-class MessageFilterBase
-{
-public:
-  typedef std::vector<std::string> V_string;
-
-  virtual ~MessageFilterBase(){}
-  virtual void clear() = 0;
-  virtual void setTargetFrame(const std::string& target_frame) = 0;
-  virtual void setTargetFrames(const V_string& target_frames) = 0;
-  virtual void setTolerance(const tf2::Duration& tolerance) = 0;
-};
-
-/**
- * \brief Follows the patterns set by the message_filters package to implement a filter which only passes messages through once there is transform data available
- *
- * The callbacks used in this class are of the same form as those used by roscpp's message callbacks.
- *
- * MessageFilter is templated on a message type.
- *
- * \section example_usage Example Usage
- *
- * If you want to hook a MessageFilter into a ROS topic:
-\verbatim
-message_filters::Subscriber<MessageType> sub(node_handle_, "topic", 10);
-tf::MessageFilter<MessageType> tf_filter(sub, tf_listener_, "/map", 10);
-tf_filter.registerCallback(&MyClass::myCallback, this);
-\endverbatim
- */
-template<class M>
-class MessageFilter : public MessageFilterBase, public message_filters::SimpleFilter<M>
-{
-public:
-  using MConstPtr = std::shared_ptr<M const>;
-  typedef ros::MessageEvent<M const> MEvent;
-  typedef boost::function<void(const MConstPtr&, FilterFailureReason)> FailureCallback;
-  typedef boost::signals2::signal<void(const MConstPtr&, FilterFailureReason)> FailureSignal;
-
-  // If you hit this assert your message does not have a header, or does not have the HasHeader trait defined for it
-  // Actually, we need to check that the message has a header, or that it
-  // has the FrameId and Stamp traits. However I don't know how to do that
-  // so simply commenting out for now.
-  //ROS_STATIC_ASSERT(ros::message_traits::HasHeader<M>::value);
-
-  /**
-   * \brief Constructor
-   *
-   * \param bc The tf2::BufferCore this filter should use
-   * \param target_frame The frame this filter should attempt to transform to.  To use multiple frames, pass an empty string here and use the setTargetFrames() function.
-   * \param queue_size The number of messages to queue up before throwing away old ones.  0 means infinite (dangerous).
-   * \param nh The NodeHandle whose callback queue we should add callbacks to
-   */
-  MessageFilter(tf2::BufferCore& bc, const std::string& target_frame, uint32_t queue_size, const ros::NodeHandle& nh)
-  : bc_(bc)
-  , queue_size_(queue_size)
-  , callback_queue_(nh.getCallbackQueue())
+  namespace filter_failure_reasons
```

Comment:
We follow the Google style of not indenting after a namespace.  Switching back to that style will be more conformant with our style, will not fail CI, and will probably reduce the amount of differences in this change.

Reply:
I give some coding style alignment with your comment, thanks.